### PR TITLE
Reorganize files to make importation easier

### PR DESCRIPTION
**Pull request by daynewlee**

### CODE REVIEW

File: .github/workflows/schema-validation.yml [hunk 1]
```diff
@@ -29,4 +29,4 @@ jobs:
             exit 1
           fi
       - name: Validate CVEs
-        run: go run -v ./validate
+        run: go run -v main.go
```
Comment (RTann) [inline .github/workflows/schema-validation.yml:32]: you can do `go run -v ./` I believe

File: main.go [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package main
 
 import (
+	validation2 "github.com/stackrox/istio-cves/validation"
```
Comment (RTann) [inline main.go:4]: move this down with the other `github.com` imports and also remove the name `validation2`

File: validation/types.go [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package main
+package validation
```
Comment (RTann) [inline validation/types.go:1]: Let's put this in another directory/package so when others use it they dont use `validate.Vuln`. Maybe something like `vuln` (not to be confused with the `vulns` directory which already exists)

Comment (daynewlee) [reply validation/types.go:1]: But isn't vuln.Vuln more confusing?

Comment (RTann) [reply validation/types.go:1]: Thinking about it from a user of this library, I think it makes more sense than `validate.Vuln`. I'd also be ok with a `types/` directory and putting it in a `types` package. Maybe let's do that

Comment (daynewlee) [reply validation/types.go:1]: > Thinking about it from a user of this library, I think it makes more sense than `validate.Vuln`. I'd also be ok with a `types/` directory and putting it in a `types` package. Maybe let's do that

That makes sense. I will do that.

File: types/validate.go [hunk 4]
```diff
@@ -1,4 +1,4 @@
-package main
+package types
```
Comment (RTann) [inline types/validate.go:1]: I think it's ok to keep this file in the `main` package and to keep it package-private (lowercase). This isn't a function someone outside this repo would need to use, so we shouldn't expose it. Also, this validation isn't really validating the `types` but rather the vulns, so I don't think it makes sense in `types/`

File: validate.go [hunk 5]
```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	"fmt"
+	"github.com/stackrox/istio-cves/types"
```
Comment (RTann) [inline validate.go:5]: put this with the other `github.com` imports

File: validate.go [hunk 6]
```diff
@@ -23,13 +24,14 @@ var (
...
-	firstPublishedCVE, err = time.Parse(TimeLayout, "2019-05-28T00:00Z")
+	firstPublishedCVE, err = time.Parse(types.TimeLayout, "2019-05-28T00:00Z")
 	if err != nil {
 		panic("Should not happen")
 	}
 }
 
-func validate(fileName string, vuln *Vuln) error {
+// Validate yaml files
+func Validate(fileName string, vuln *types.Vuln) error {
```
Comment (RTann) [inline validate.go:34]: `validate` instead of `Validate`. No need to make this accessible to other packages

**Review by RTann — CHANGES_REQUESTED** [review]

**Review by RTann — APPROVED** [review]

**Pull request merged**